e2e/pages/administration/jobs/background/background.spec.js

```diff
@@ -0,0 +1,30 @@
+import LoginComponent from '../../../../components/login/login.po';
+import BackgroundPage from './background.po';
+import Hooks from '../../../../utilities/hooks';
+
+let background, hooks, loginComponent;
+
+beforeEach(async () => {
+    loginComponent = new LoginComponent();
+    background = new BackgroundPage();
+    hooks = new Hooks();
+    await hooks.open('#/search');
+});
+
+describe('ONC-Staff user can see', () => {
```

Comment:
When in `describe` blocks, the phrasing I usually use is something where the text in the describe phrase should explain the setup, so here it'd be something like 
```suggestion
describe('when an ONC-Staff user is logged in', () => {
```